pandas/tests/groupby/test_groupby.py

```diff
@@ -4304,6 +4251,192 @@ def test_cummin_cummax(self):
             tm.assert_series_equal(expected, result)
 
 
+class TestCounting(tm.TestCase):
```

Comment:
perfect you moved them! even better, can you move to a separate file

``pandas/tests/groupby/test_counting.py`` (cumcount & ngroup). If you find other methods that are very relevant ok (note I wouldn't move .count()).